services/agent/src/simcore_service_agent/services/backup.py

```diff
@@ -148,6 +162,8 @@ async def _store_in_s3(
         volume_details.labels, s3_path, r_clone_ls_output, volume_name
     )
 
+    await _ensure_permissions_on_source_dir(source_dir)
```

Comment:
and this is not going to trigger other errors elsewhere?
do you have somewhere where these permissions are documented?? that would be quite helpful the day we want to tackle that.

Reply:
ok